packages/labs/analyzer/src/test/vanilla-element/jsdoc_test.ts

```diff
@@ -188,147 +188,19 @@ for (const lang of languages) {
-    assert.ok(element.isCustomElementDeclaration());
-    assert.equal(
-      element.description,
-      `TaggedDescription description. Lorem ipsum dolor sit amet, consectetur
-adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
-aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-nisi ut aliquip ex ea commodo consequat.`
-    );
-    assert.equal(element.summary, `TaggedDescription summary.`);
-    assert.equal(element.deprecated, `TaggedDescription deprecated message.`);
-  });
-
-  test('untagged description', ({getModule}) => {
-    const element = getModule('element-a').getDeclaration(
-      'UntaggedDescription'
-    );
-    assert.ok(element.isCustomElementDeclaration());
-    assert.equal(
-      element.description,
-      `UntaggedDescription description. Lorem ipsum dolor sit amet, consectetur
-adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna
-aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris
-nisi ut aliquip ex ea commodo consequat.`
-    );
-    assert.equal(element.summary, `UntaggedDescription summary.`);
-    assert.equal(element.deprecated, `UntaggedDescription deprecated message.`);
-  });
-
-  // Fields
-
-  test('field1', ({getModule}) => {
-    const element = getModule('element-a').getDeclaration('ElementA');
-    assert.ok(element.isClassDeclaration());
-    const member = element.getField('field1');
-    assert.ok(member?.isClassField());
-    assert.equal(
-      member.description,
-      `Class field 1 description\nwith wraparound`
-    );
-    assert.equal(member.default, `'default1'`);
-    assert.equal(member.privacy, 'private');
-    assert.equal(member.type?.text, 'string');
-  });
-
-  test('field2', ({getModule}) => {
-    const element = getModule('element-a').getDeclaration('ElementA');
-    assert.ok(element.isClassDeclaration());
-    const member = element.getField('field2');
-    assert.ok(member?.isClassField());
-    assert.equal(member.summary, `Class field 2 summary\nwith wraparound`);
-    assert.equal(
-      member.description,
-      `Class field 2 description\nwith wraparound`
-    );
-    assert.equal(member.default, undefined);
-    assert.equal(member.privacy, 'protected');
-    assert.equal(member.type?.text, 'string | number');
-  });
-
-  test('field3', ({getModule}) => {
-    const element = getModule('element-a').getDeclaration('ElementA');
-    assert.ok(element.isClassDeclaration());
-    const member = element.getField('field3');
-    assert.ok(member?.isClassField());
-    assert.equal(
-      member.description,
-      `Class field 3 description\nwith wraparound`
-    );
-    assert.equal(member.default, undefined);
-    assert.equal(member.privacy, 'public');
-    assert.equal(member.type?.text, 'string');
-    assert.equal(member.deprecated, true);
-  });
-
-  test('field4', ({getModule}) => {
+  test('basic class analysis', ({getModule}) => {
```

Comment:
Moved all the class-specific stuff to `classes_test.js`, and just left a basic class analysis test here.